apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts

```diff
@@ -95,8 +95,8 @@ async function calculateTokensStats(
   }, 0);
 
   const tokenPromises = poolTokenData.map(async (token, idx) => {
-    const tokenPrice = await getTokenPriceByRound(
-      Round.getRoundByNumber(roundId),
+    const tokenPrice = await getTokenPriceByDate(
+      Round.getRoundByNumber(roundId).endDate,
```

Comment:
🤔 not sure I got why change this -- if anything we should change the API to get a start/end date instead of a fixed date, no?